src/glyphs.cpp

```diff
@@ -548,10 +548,6 @@ void RenderSDF(glyph_info &glyph,
         return;
     }
 
-    FT_Glyph ft_glyph = nullptr;
-    ft_glyph_guard glyph_guard(&ft_glyph);
-    if (FT_Get_Glyph(ft_face->glyph, &ft_glyph)) return;
```

Comment:
I was getting mixed up before, the earlier `FT_Load_Glyph` call is what loads the glyph into the glyph slot on `FT_Face`, so yea I think this is unnecessary.